AK/BitStream.h

```diff
@@ -141,7 +140,8 @@ class InputBitStream final : public InputStream {
     }
 
 private:
-    Optional<u8> m_next_byte;
+    u8 m_next_byte { 0 };
```

Comment:
What was the perf impact of this change? The commit message makes it unclear if it helped or not?

Reply:
Looking back, I think I'll throw this out because it didn't help measurably.